packages/server/observability/logging.ts

```diff
@@ -6,7 +6,7 @@ const { getLogger, extendLoggerComponent } = Observability
 
 export const logger = getLogger(
   process.env.LOG_LEVEL || 'info',
-  process.env.LOG_PRETTY === 'true'
+  process.env.LOG_PRETTY === 'true' && !process.env.FORCE_NO_PRETTY
```

Comment:
on node 22 pino pretty causes debugger to freeze on startup sometime. i don't want to change the env vars in the `yarn dev` command every time, so it's cleaner to just introduce an extra flag that disables it forcefully irregardless of what LOG_PRETTY is so I can just do `FORCE_NO_PRETTY=1 yarn dev`